server/controllers/account.controllers.js

```diff
@@ -29,7 +29,8 @@ module.exports = {
       // console.log('LISTINGS');
       models.listings.get(owner_id, (err, data) => {
         if (err) {
-          res.status(500).send(err);
+          console.log('LISTINGS ERROR', err);
+          res.status(500).send(err, err);
```

Comment:
super minor, but you are sending err twice!

Reply:
fixed!